tests/php/ORM/UnionListTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace SilverStripe\ORM\Tests;
+
+use SilverStripe\Dev\Debug;
+use SilverStripe\Core\Convert;
+use SilverStripe\ORM\UnionList;
+use SilverStripe\ORM\DB;
+use SilverStripe\ORM\Filterable;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\Tests\DataObjectTest\EquipmentCompany;
+use SilverStripe\ORM\Tests\DataObjectTest\Fan;
+use SilverStripe\ORM\Tests\DataObjectTest\Player;
+use SilverStripe\ORM\Tests\DataObjectTest\Sortable;
+use SilverStripe\ORM\Tests\DataObjectTest\SubTeam;
+use SilverStripe\ORM\Tests\DataObjectTest\Team;
+use SilverStripe\ORM\Tests\DataObjectTest\TeamComment;
+use SilverStripe\ORM\Tests\DataObjectTest\ValidatedObject;
```

Comment:
Yup, was going to remove them at the end. IDE doesn't automatically add "use". Will need to look into that.